server/controllers/validate-token-controller.js

```diff
@@ -19,7 +19,7 @@ const validateToken = (req, res) => {
       const validateTokenUrl = reverse("validate_auth_token", getSlug(conf));
       const timeout = conf.timeout * 1000;
       let {token} = req.body;
-      if (req.body.session === "false") {
+      if (req.body.session === "false" && token.includes(".")) {
```

Comment:
why is this needed?

Reply:
When the cookie is used then it was signed from the express backend but if we set the token directly then it must not be signed, So, skipping the step of signing token if it does not include `.`